opennlp-tools/src/main/java/opennlp/tools/postag/POSTaggerFactory.java

```diff
@@ -95,8 +83,19 @@ public POSTaggerFactory(byte[] featureGeneratorBytes, final Map<String, Object>
     this.posDictionary = posDictionary;
   }
 
+
+  // reduced visibility to ensure deprecation is respected in future versions
+  @Deprecated
+  POSTaggerFactory(Dictionary ngramDictionary, TagDictionary posDictionary) {
+    this.init(ngramDictionary, posDictionary);
+
+    // TODO: This could be made functional by creating some default feature generation
+    // which uses the dictionary ...
```

Comment:
:ok_man:

Reply:
This comment was there previously. Maybe, I just moved it. So no actual change intended here.